packages/block-library/src/form/edit.js

```diff
@@ -130,6 +130,8 @@ const Edit = ( { attributes, setAttributes, clientId } ) => {
 			{ submissionMethod !== 'email' && (
 				<InspectorControls group="advanced">
 					<SelectControl
+						// TODO: Switch to `true` (40px size) if possible
+						__next40pxDefaultSize={ false }
```

Comment:
(excuses if this has already been asked) Do we have a place where we track all the occurrences that need updating and are exception to the lint rule?

Reply:
I'm assuming you're asking about discoverability. A few ways:

- I'm leaving a separate checklist item in #63871 so we know there are remaining violations.
- We can search the TODO comment string.
- We can move this component to the stricter lint rule in the eslintrc, which doesn't allow `__next40pxDefaultSize={ false }`. That would surface all the remaining violations.

Reply:
Makes sense, thank you!